src/main/java/uk/gov/hmcts/sptribs/ciccase/model/CicCase.java

```diff
@@ -127,8 +129,78 @@ public class CicCase {
         label = "What is their contact preference?",
         access = {DefaultAccess.class, CaseworkerWithCAAAccess.class}
     )
-    private ContactPreferenceType contactDetailsPreference;
-    private ApplicantCICDetails applicantCICDetails;
-    private RepresentativeCICDetails representativeCICDetails;
+    private ContactPreferenceType contactDetailsPrefrence;
+
+    @CCD(
+        label = "Full Name",
+        access = {DefaultAccess.class, CaseworkerWithCAAAccess.class}
+    )
+    private String applicantFullName;
+
+    @CCD(label = "Address")
+    private AddressGlobalUK applicantAddress;
+
+    @CCD(
+        label = "Phone number",
+        regex = "^[0-9 +().-]{9,}$"
+    )
+    private String applicantPhoneNumber;
+
+    @CCD(
+        label = "Email address",
+        typeOverride = Email
+    )
+    private String applicantEmailAddress;
+
+    @CCD(
+        label = "Date of birth",
+        access = {DefaultAccess.class, CaseworkerWithCAAAccess.class}
+    )
+    @JsonFormat(pattern = "yyyy-MM-dd")
+    private LocalDate applicantDateOfBirth;
+
+    @CCD(
+        label = "What is their contact preference?",
+        access = {DefaultAccess.class, CaseworkerWithCAAAccess.class}
+    )
+    private ContactPreferenceType applicantContactDetailsPreference;
+
+    @CCD(
+        label = "Full Name",
+        access = {DefaultAccess.class, CaseworkerWithCAAAccess.class}
+    )
+    private String representativeFullName;
+
+    @CCD(label = "Address")
+    private AddressGlobalUK representativeAddress;
+
+    @CCD(
+        label = "Phone number",
+        regex = "^[0-9 +().-]{9,}$"
+    )
```

Comment:
As above, should write a unit test for this regex

Reply:
@osheaf01 , As discussed on standup, it is not possible to test regex using unit tests as the validation happens by CCD component.